azure-cli.pyproj

```diff
@@ -117,6 +115,7 @@
     <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\commands.py" />
     <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\credential.py" />
     <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\custom.py" />
+    <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\multiarch.py"/>
```

Comment:
Can you double check if you can revert other changes in this file besides this line?